rest/src/main/java/org/jboss/pnc/rest/provider/ScheduleConflictExceptionMapper.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.rest.provider;
+
+import org.jboss.pnc.spi.exception.ScheduleConflictException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+import java.lang.invoke.MethodHandles;
+import org.jboss.pnc.dto.response.ErrorResponse;
+
+@Provider
+public class ScheduleConflictExceptionMapper implements ExceptionMapper<ScheduleConflictException> {
+
+    private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    @Override
+    public Response toResponse(ScheduleConflictException e) {
+        logger.warn("A ScheduleConflict error occurred when scheduling Build", e);
```

Comment:
```suggestion
        logger.warn("A ScheduleConflict error occurred when scheduling Build. {}", e.getMessatge());
```
I wouldn't print the entire stacktrace because this exception comes from just one place.